sdk/src/Handlers/System.Net/AWSXRayRecorder.Handlers.System.Net.csproj

```diff
@@ -30,7 +30,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.3.25.1" />
+    <PackageReference Include="AWSSDK.Core" Version="3.5.1.23" />
```

Comment:
Same here. Probably we can remove this dependency

Reply:
Same as above